go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -82,6 +82,7 @@ const (
 	// in effect on the source when it was created:
 	//   https://github.com/mysql/mysql-server/blob/3290a66c89eb1625a7058e0ef732432b6952b435/client/mysqldump.cc#L795-L818
 	SQLMode          = "NO_AUTO_VALUE_ON_ZERO"
```

Comment:
Now that we have `StrictSQLMode`, probably worth renaming this to ~ `PermissiveSQLMode`

Reply:
That's a good idea! I'll leave this for another PR as I'm keen to keep this PR as small as possible.